searx/settings.yml

```diff
@@ -1720,6 +1720,20 @@ engines:
       require_api_key: false
       results: HTML
 
+  - name: petal images
```

Comment:
```suggestion
  - name: petalsearch images
```

Reply:
For reference: https://github.com/searxng/searxng/blob/51ba817e06bb15ca1768010d6873d1d7bf48b0b6/searx/settings.yml#L1723